ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/InsertsStreamHandler.java

```diff
@@ -159,11 +160,17 @@ private void handleArgs(final Buffer buff) {
 
     private Void handleInsertSubscriberException(final Throwable t,
```

Comment:
The logic here is similar to the failure handler in QueryStreamHandler, perhaps could consider combining it?

Reply:
Done -- refactored into ServerUtils.